src/ATLTileCalTBSensDet.cc

```diff
@@ -115,8 +112,8 @@ G4bool ATLTileCalTBSensDet::ProcessHits( G4Step* aStep, G4TouchableHistory* ) {
 
     //Add hit energy 
     //
-    hit->AddE( edep );
-    hit->AddS( sdep );
+    hit->AddEdep(edep);
+    hit->AddSdep(aStep->GetPreStepPoint()->GetGlobalTime(), sdep_up, sdep_down);
```

Comment:
Is getting the time via `PreStep` correct?

Reply:
Yes, it is fine.